berkdb/rep/rep_record.c

```diff
@@ -5192,7 +5193,7 @@ bad_resize:	;
 		goto err;
 #endif
 	}
-
+	gbl_rep_lockid = lockid;
```

Comment:
YES .. parallel rep can have multiple outstanding lockers, but only one of them will be acquiring locks at any given time ..